src/sage/modular/modform_hecketriangle/graded_ring_element.py

```diff
@@ -1580,7 +1582,7 @@ def _q_expansion_cached(self, prec, fix_d, subs_d, d_num_prec, fix_prec = False)
         d = self.parent().get_d(fix_d, d_num_prec)
         q = self.parent().get_q(prec, fix_d, d_num_prec)
 
-        qexp = sum([(qexp.coefficients()[m]).subs(d=d) * q**qexp.exponents()[m] for m in range(len(qexp.coefficients()))])
+        qexp = sum([((qexp.coefficients()[m]).subs(d=d)) * q**qexp.exponents()[m] for m in range(len(qexp.coefficients()))])
```

Comment:
This extra set of parentheses seems unless (and IMO, makes it harder to parse).

Reply:
I don't know what was the intention of the original author, but yes, I think we can remove them.